### PR TITLE
[MOOV-2204]: Added payout metric tabs to dashboard

### PR DESCRIPTION
**Pull request by agranillonf**
Added payout metric tabs to the payouts dashboard, with corresponding filter actions.

Also added Payout Payee sorting (won't work until MoneyMoov API PR #1211 is merged).

### CODE REVIEW

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/53.

**Review by donalnofrixion — APPROVED** [review]

**Review by pabloi-nf — CHANGES_REQUESTED** [review]
I see 2 tabs with 0, but with entries

image
image

**Review by pabloi-nf — APPROVED** [review]

**Comment by agranillonf** [comment]
@pabloi-nf MoneyMoov API has been deployed to develop, so this PR can now be fully tested.

File: apps/business/src/pages/payouts/page.tsx [hunk 1]
```diff
@@ -33,7 +40,17 @@ const PayoutsPage = () => {
     return <Navigate to={location.pathname.replace(`/${payoutId}`, '').replace(`/${result}`, '')} />
   }
 
-  return <>{merchant && <PayoutDashboard merchantId={merchant.id} apiUrl={NOFRIXION_API_URL} />}</>
+  return (
+    <>
```
Comment (Publicker) [inline apps/business/src/pages/payouts/page.tsx:44]: We could get rid of this Fragment.

**Review by Publicker — APPROVED** [review]

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/53.

**Comment by agranillonf** [comment]
@donalnofrixion @Publicker re-requesting your approvals since the solution to Ticket #2235 (Payee sorting) is now included in this PR (won't work until MoneyMoov API PR #1211 is merged).

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/53.

**Review by donalnofrixion — APPROVED** [review]

**Review by pabloi-nf — APPROVED** [review]

**Review by Publicker — APPROVED** [review]

**Comment by agranillonf** [comment]
@pabloi-nf Re-requesting your review since you can now test Payee sorting.

**Review by pabloi-nf — APPROVED** [review]

**Comment by sipsorcery** [comment]
Latest changes for this PR can be tested at https://business-dev.nofrixion.com/53.

**Pull request merged**